tcms/testplans/static/testplans/js/get.js

```diff
@@ -686,6 +686,26 @@ function toolbarEvents(testPlanId, permissions) {
 
          window.location.assign(`/cases/clone?case=${selectedCases.join('&case=')}`);
     });
+
+    $('#testplan-toolbar-newrun').click(function(ev) {
```

Comment:
`$(this).parents('.dropdown').toggleClass('open');` & `return false` down below. See lines 577-580